pkg/segment/aggregations/segaggs.go

```diff
@@ -1580,7 +1580,11 @@ func performStatisticColRequestOnMeasureResults(nodeResult *structs.NodeResult,
 	resTotal := uint64(0)
 	if letColReq.StatisticColRequest.StatisticOptions.ShowPerc {
 		for _, bucketHolder := range nodeResult.MeasureResults {
-			resTotal += bucketHolder.MeasureVal[countName].(uint64)
+			value, exists := bucketHolder.MeasureVal[countName]
+			if !exists {
+				continue
```

Comment:
if value does not exist and this is a new bucket key, then we should create an entry in the bucket for this new key

Reply:
actually ignore this comment, we are just counting the resTotal if there is value for this bucket key

Reply:
But we are actually accessing the same key from the same map below. So it might be better to create an entry here?